cmds/exp/df/df.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2021 the u-root Authors. All rights reserved
```

Comment:
df is already a well-known command in Linux. Perhaps call this something else like `lsfabric`.